lib/config/genconfig/hadeployconfig.go

```diff
@@ -15,6 +15,7 @@ const (
 	AWS_S3                        = "AWS S3"
 	MINIO                         = "Minio"
 	OBJECT_STORE                  = "Object Store"
+	GOOGLE_BUCKET                 = "Google Storage Bucket"
```

Comment:
Should this be like this?
```suggestion
	GOOGLE_BUCKET                 = "Google Cloud Storage"
```